### DISCUSSION TITLE
Is airflow REST API (stable) HA?

### DISCUSSION THREAD

**Discussion by HashSalmon** [apache/airflow#15553] [Q&A] [answered]
Hey, I am thinking to use k8s executor and move airflow on the k8s cluster.  My question is which part of the service handles the rest api in airflow? Is that service HA? Thanks

**Comment by potiuk** [comment, marked as answer]
Webserver. It is stateless, so yes, you can run several webservers and make it HA.

**Reply by HashSalmon** [reply]
thank you!
